### DISCUSSION TITLE
Integration with MDXP or other slide presentation library?

### DISCUSSION THREAD

**Discussion by zaphod72** [code-hike/codehike#287] [Q&A] [answered]
Is it possible to run Code Hike in MDXP https://0phoff.github.io/MDXP/ ?
I'm not very experienced with React so wondering if you have any advice in trying this out.

Or is there another slide presentation library I can use with Code Hike?

TIA

**Comment by pomber** [comment, marked as answer]
I haven't tried MDXP but it looks like it uses [MDX v1](https://github.com/0phoff/MDXP/blob/master/package.json#L36) and Code Hike needs MDX v2. 

If you want the code transitions for your slides take a look at [Code Surfer](https://github.com/pomber/code-surfer).
